### DISCUSSION TITLE
Issue with Sending Emails via SMTP using SSL/TLS (Port 465)

### DISCUSSION THREAD

**Discussion by Le-Merovingien** [supabase/supabase#31046] [Questions]
Hello,

I am encountering an issue with the SMTP configuration in Supabase for sending emails. Here are the details:

My SMTP server is configured to use SSL/TLS on port 465. This configuration works perfectly with other clients, such as Thunderbird and a Python script using SMTP_SSL.
When I try to use this same configuration in Supabase, I receive the following error:
Failed to invite user: upstream request timeout
I have also tested using STARTTLS on port 587 (as an alternative). While this works outside Supabase, I prefer to use SSL/TLS on port 465 for security and compatibility with my other tools.

Technical Details:

    Port Used: 465
    Secure Connection: SSL/TLS
    Authentication: Yes (requires username and password)
    The server is running under Docker Mailserver with valid SSL/TLS certificates.

Questions:

    Does Supabase support SMTP connections using SSL/TLS on port 465? If so, how can I configure this in the interface?
    If not supported, is there a workaround to use SSL/TLS on port 465 with Supabase?

I am happy to provide additional information or perform tests if needed.

Thank you for your assistance.

**Comment by leszko11** [comment]
Hey!

Any updates on this? 🙏
